staging/src/k8s.io/apiextensions-apiserver/pkg/apihelpers/helpers_test.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos - file modified.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// File modified by cherrypick from kubernetes on 02/25/2021
+package apihelpers
+
+import (
+	"testing"
+
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
Why v1beta1, instead of v1, is used here?

Reply:
Helpers.go and helpers_test.go is added before we promote to CRD V1. there are separated PR to promote helpers to CRD V1 in community: https://github.com/kubernetes/kubernetes/pull/84005. In order to reduce the complication of this PR, so not included community PR#84005 in this PR. we can think about if necessary in the future.